tests/CircuitBreakerWorkflowTest.php

```diff
@@ -100,6 +100,29 @@ public function testOnceInHalfOpenModeServiceIsFinallyReachable(CircuitBreaker $
         $this->assertTrue($circuitBreaker->isClosed());
     }
 
+    /**
+     * The Circuit Breaker can be isolated, once its done it remains
+     * Open and so on only fallback responses will be sent.
+     *
+     * @dataProvider getCircuitBreakers
+     */
+    public function testOnceCircuitBreakerIsIsolatedNoTrialsAreDone(CircuitBreaker $circuitBreaker): void
```

Comment:
maybe add `expect` on the mock, or assert the number of calls in another way would be a good thing for this test?

Reply:
yes, I need to complete the test suite